This patch was created in-house. The author of cdrtools thinks this is
a compiler issue so the patch will not be submitted upstream.

--- cdrtools-3.01/libedc/libedc.mk	2010-06-23 13:48:26.000000000 -0700
+++ cdrtools-3.01/libedc/libedc.mk.new	2014-06-05 04:12:22.776598049 -0700
@@ -22,7 +22,7 @@
 _XARCH_OPT=	$(OARCH:%cc64=$(SUNPROCOPT64))
 XARCH_OPT=	$(_XARCH_OPT:%cc=$(XARCH_GEN))
 
-SUNPROCOPTOPT=	-fast $(XARCH_OPT)
+SUNPROCOPTOPT=	$(XARCH_OPT)
 GCCOPTOPT=	-O3  -fexpensive-optimizations
 #
 CFILES=		edc_ecc.c
--- cdrtools-3.01/libedc/libedc_dec.mk	2010-06-23 13:48:26.000000000 -0700
+++ cdrtools-3.01/libedc/libedc_dec.mk.new	2014-06-05 04:12:22.789134217 -0700
@@ -18,7 +18,7 @@
 _XARCH_OPT=	$(OARCH:%cc64=$(SUNPROCOPT64))
 XARCH_OPT=	$(_XARCH_OPT:%cc=$(XARCH_GEN))
 
-SUNPROCOPTOPT=	-fast $(XARCH_OPT)
+SUNPROCOPTOPT=	$(XARCH_OPT)
 GCCOPTOPT=	-O3  -fexpensive-optimizations
 #
 CFILES=		edc_ecc_dec.c
--- cdrtools-3.01/libedc/libedc_dec_p.mk	2010-06-23 13:48:26.000000000 -0700
+++ cdrtools-3.01/libedc/libedc_dec_p.mk.new	2014-06-05 04:12:22.853051047 -0700
@@ -21,7 +21,7 @@
 _XARCH_OPT=	$(OARCH:%cc64=$(SUNPROCOPT64))
 XARCH_OPT=	$(_XARCH_OPT:%cc=$(XARCH_GEN))
 
-SUNPROCOPTOPT=	-fast $(XARCH_OPT)
+SUNPROCOPTOPT=	$(XARCH_OPT)
 GCCOPTOPT=	-O3  -fexpensive-optimizations
 #
 CFILES=		edc_ecc_dec.c
--- cdrtools-3.01/libedc/libedc_p.mk	2010-06-23 13:48:26.000000000 -0700
+++ cdrtools-3.01/libedc/libedc_p.mk.new	2014-06-05 04:12:22.866249716 -0700
@@ -25,7 +25,7 @@
 _XARCH_OPT=	$(OARCH:%cc64=$(SUNPROCOPT64))
 XARCH_OPT=	$(_XARCH_OPT:%cc=$(XARCH_GEN))
 
-SUNPROCOPTOPT=	-fast $(XARCH_OPT)
+SUNPROCOPTOPT=	$(XARCH_OPT)
 GCCOPTOPT=	-O3  -fexpensive-optimizations
 #
 CFILES=		edc_ecc.c
--- cdrtools-3.01/libedc/shledc.mk	2010-06-23 13:48:26.000000000 -0700
+++ cdrtools-3.01/libedc/shledc.mk.new	2014-06-05 04:12:22.945694830 -0700
@@ -23,7 +23,7 @@
 _XARCH_OPT=	$(OARCH:%cc64=$(SUNPROCOPT64))
 XARCH_OPT=	$(_XARCH_OPT:%cc=$(XARCH_GEN))
 
-SUNPROCOPTOPT=	-fast $(XARCH_OPT)
+SUNPROCOPTOPT=	$(XARCH_OPT)
 GCCOPTOPT=	-O3  -fexpensive-optimizations
 #
 CFILES=		edc_ecc.c
--- cdrtools-3.01/libedc/shledc_dec.mk	2010-06-23 13:48:26.000000000 -0700
+++ cdrtools-3.01/libedc/shledc_dec.mk.new	2014-06-05 04:12:23.010548133 -0700
@@ -19,7 +19,7 @@
 _XARCH_OPT=	$(OARCH:%cc64=$(SUNPROCOPT64))
 XARCH_OPT=	$(_XARCH_OPT:%cc=$(XARCH_GEN))
 
-SUNPROCOPTOPT=	-fast $(XARCH_OPT)
+SUNPROCOPTOPT=	$(XARCH_OPT)
 GCCOPTOPT=	-O3  -fexpensive-optimizations
 #
 CFILES=		edc_ecc_dec.c
--- cdrtools-3.01/libmdigest/mdigest.mk	2010-06-23 13:49:56.000000000 -0700
+++ cdrtools-3.01/libmdigest/mdigest.mk.new	2014-06-05 04:13:04.129363504 -0700
@@ -22,7 +22,7 @@
 _XARCH_OPT=	$(OARCH:%cc64=$(SUNPROCOPT64))
 XARCH_OPT=	$(_XARCH_OPT:%cc=$(XARCH_GEN))
 
-SUNPROCOPTOPT=	-fast $(XARCH_OPT)
+SUNPROCOPTOPT=	$(XARCH_OPT)
 GCCOPTOPT=	-O3  -fexpensive-optimizations
 #
 include		Targets
--- cdrtools-3.01/libmdigest/shlmdigest.mk	2010-06-23 13:49:56.000000000 -0700
+++ cdrtools-3.01/libmdigest/shlmdigest.mk.new	2014-06-05 04:12:48.435764341 -0700
@@ -23,7 +23,7 @@
 _XARCH_OPT=	$(OARCH:%cc64=$(SUNPROCOPT64))
 XARCH_OPT=	$(_XARCH_OPT:%cc=$(XARCH_GEN))
 
-SUNPROCOPTOPT=	-fast $(XARCH_OPT)
+SUNPROCOPTOPT=	$(XARCH_OPT)
 GCCOPTOPT=	-O3  -fexpensive-optimizations
 #
 include		Targets
